src/main/java/tempest/ui/components/ModuleDropDown.java

```diff
@@ -1,20 +1,18 @@
 package tempest.ui.components;
 
-import java.util.ArrayList;
-
-import javax.swing.DefaultComboBoxModel;
-import javax.swing.JComboBox;
-
 import tempest.Module;
 import tempest.State;
 
+import javax.swing.*;
+import java.util.ArrayList;
+
 public class ModuleDropDown {
 
     private static DefaultComboBoxModel<Object> model;
 
     /**
-     * This constructor should be used by all classes that need a ModuleDropDown
-     * instance
+     * This constructor should be used by all classes that
+     * need a ModuleDropDown instance
      */
     public ModuleDropDown() {
```

Comment:
Should we make this closer to a Singleton pattern, or is it fine as is?

Reply:
I think its fine - it works